packages-content-model/roosterjs-content-model-dom/lib/modelToDom/handlers/handleEntity.ts

```diff
@@ -23,7 +24,23 @@ export const handleEntityBlock: ContentModelBlockHandler<ContentModelEntity> = (
 
     applyFormat(wrapper, context.formatAppliers.entity, entityFormat, context);
 
-    refNode = reuseCachedElement(parent, wrapper, refNode);
+    const isCursorAroundEntity =
+        context.addDelimiterForEntity &&
+        wrapper.style.display == 'inline-block' &&
+        wrapper.style.width == '100%';
+    const isContained = wrapper.parentElement?.classList.contains('blockEntityContainer');
+    const elementToReuse = isContained && isCursorAroundEntity ? wrapper.parentElement! : wrapper;
```

Comment:
Is it possible parentElement is the same with parent, or event parent.parent? If that really happens, what can we do?

Reply:
added an additional check for this. if parent==element.parentElement we should not try to reuse the container and use the entity. Probably no going to happen often.